### PR TITLE
Toggle for touch only pan/select

### PR DESCRIPTION
**Pull request by ccai1**
This is an attempt at implementing #39 to allow disabling or customizing for touch mode, especially for tablets. This change adds a button (below eraser) that toggles between restricting touch to only pan/select and allowing touch for all functions, including drawing. The default mode is touch only pan/select.

To help create a toggle, I tried to follow the code for "Toggle grid/graph paper," so I'm not sure if some lines are extraneous 🙂 (The button toggles the truth value of variable `currentTouch` and drawing events for pen, line, ellipse, rect, etc. are stopped if `currentTouch` equals `false` and `pointerType` equals `touch`). This feature was tested on a non-touch-enabled computer and a tablet with stylus.

Some concerns:
- It may be confusing for a computer user with no touch screen to use the toggle, because it doesn't change any visible features for them.
- Would it be useful to make a popup notification for the toggle if user is on tablet mode?

### CODE REVIEW

File: lib/rooms.coffee [hunk 1]
```diff
@@ -22,3 +23,9 @@ Meteor.methods
     data = checkRoom room
     Rooms.update room,
       $set: grid: not data.grid
+
+  roomTouchToggle: (room) ->
+    check room, String
+    data = checkRoom room
+    Rooms.update room,
+      $set: touch: not data.touch
```
Comment (edemaine) [inline lib/rooms.coffee:31]: I think touch enable/disable should be a local variable, not global to the room. The way you have it set up, toggling the button will change it for everyone in the same room!

File: client/main.coffee [hunk 2]
```diff
@@ -250,13 +256,13 @@ tools =
...
     up: (e) ->
-      return unless pointers[e.pointerId]
+      return unless pointers[e.pointerId] && !restrictTouch e
       undoableOp
         type: 'new'
         obj: Objects.findOne pointers[e.pointerId]
       delete pointers[e.pointerId]
     move: (e) ->
-      return unless pointers[e.pointerId]
+      return unless pointers[e.pointerId] && !restrictTouch e
```
Comment (edemaine) [inline client/main.coffee:265]: Stylistically, I'd split this up into two lines, where the new line is `return if restrictTouch e`

File: lib/rooms.coffee (hunk 1 above)
Comment (ccai1) [reply lib/rooms.coffee]: Should be fixed with new commit? Thanks for the feedback!

File: client/main.coffee (hunk 2 above)
Comment (ccai1) [reply client/main.coffee]: Fixed in the new commit!

**Comment by edemaine** [comment]
@ccai1 Thanks again for making this happen! I did some testing and it does seem like pretty good behavior, at least as a first step. I also changed the default behavior to *allow* touch input, which seems pretty important.

I did a bunch of revisions/cleanup to this PR (directly on your branch). Please have a look, both to learn about stylistic improvements/consistency and to see that you agree e.g. with the new help text. Then I'll merge.

**Comment by ccai1** [comment]
@edemaine Looks good to me! I think the description, position, and icon of the toggle makes its purpose clearer to the user now. Thanks for the revisions.

**Pull request merged**